src/gui/advancedsettings.cpp

```diff
@@ -485,6 +489,19 @@ void AdvancedSettings::loadAdvancedSettings()
     // multiple connections per IP
     m_checkBoxMultiConnectionsPerIp.setChecked(session->multiConnectionsPerIpEnabled());
     addRow(MULTI_CONNECTIONS_PER_IP, tr("Allow multiple connections from the same IP address"), &m_checkBoxMultiConnectionsPerIp);
+#if (LIBTORRENT_VERSION_NUM >= 10206)
+    // Validate HTTPS trackers
+    m_checkBoxValidateHTTPSTrackers.setChecked(session->validateHTTPSTrackers());
+    addRow(VALIDATE_HTTPS_TRACKERS, tr("Validate HTTPS Trackers"), &m_checkBoxValidateHTTPSTrackers);
+    // SSL version
+#if (BOOST_VERSION >= 106900)
+    m_comboBoxSSLVersion.addItems({tr("TLSv1.1"), tr("TLSv1.2"), tr("TLSv1.3")});
```

Comment:
I don't think this needs `tr()`.
And I would replace the `v` with a whitespace.